settings.py

```diff
@@ -223,7 +225,7 @@
     "POSTMARK_SERVER_TOKEN": os.environ.get('POSTMARK_SERVER_TOKEN', 'postmark-server-token-placeholder'),
 }
 
-if os.environ.get('DEPLOYMENT_TYPE') == 'prod':
+if os.environ.get('DEPLOYMENT_TYPE') in {'prod', 'dev'}:
```

Comment:
this condition was put in to differentiate between real deployed seqr and running locally. Since we are running all our security scans on dev, we want the dev deployment to behave like the prod one when it comes to security

Reply:
Should we also add `staging` environment which is required the same as the `prod`? So that, the `dev` can be used for demo and testing.

Reply:
no, because these are actual values that exist in our deployment code and we don't have a staging or any near term plans to make one. If we add a staging environment, we should update the code then but adding impossible. values just makes the code harder to maintain